_blog-src/_posts/2016-01-11-release-1.2.markdown

```diff
@@ -1,6 +1,6 @@
 ---
 layout: post
-title:  "Finall!  Apiman 1.2.0.Final is released!"
+title:  "Final!  Apiman 1.2.0.Final is released!"
```

Comment:
In retrospect, I think this should have been 'Finally!' not 'Final!' right @EricWittmann ? Let me know and I'll update it.